cli/injector/disk_pressure.go

```diff
@@ -46,6 +46,11 @@ var diskPressureCmd = &cobra.Command{
 				log.Fatalw("error initializing the disk pressure injector", "error", err)
 			}
 
+			if inj == nil {
+				log.Infow("Skipping current injector...")
```

Comment:
```suggestion
```

I don't think this log line is useful as it does not give any interesting context (the warn log on injector creation should be enough)

Reply:
I've added a comment instead for future context as to why we continue here.